packages/visualizations/app/styles/navi-visualizations/components/number-format-dropdown.less

```diff
@@ -4,6 +4,20 @@
  */
 
 .number-format-dropdown {
+  align-self: center;
+
+  &__trigger {
+    padding: 0 5px;
+  }
+
+  &__trigger:hover {
+    background-color: @navi-gray-300;
```

Comment:
Can we use a lighter gray: `@navi-gray-200:`

Reply:
I tried it, but it looks worse than 300. And 300 is also used for export dropdown, so it is better to use 300 here